src/index.html

```diff
@@ -23,8 +23,8 @@
       data-qa="big"
     >
       <label
-        for="searchOne"
-        class="label-image"
+        for="search--big"
```

Comment:
The modifiers are used only for CSS classes not for attributes or ids, rename this `for` into just `search-big`